cfgov/jinja2/v1/_includes/blocks/heading.html

```diff
@@ -1,6 +1,6 @@
 <{{ value.level -}}
     {%- if value.icon -%}
-        {{- ' class="cf-icon cf-icon-' ~ value.icon ~ '"' -}}
+        {{ svg_icon(value.icon) }}
     {%- endif -%}
 >
```

Comment:
The SVG needs to be moved outside the opening heading tag here.

Reply:
How would it be sized correctly?